atomic_reactor/plugins/post_pulp_pull.py

```diff
@@ -84,4 +84,9 @@ def run(self):
                        image_id)
         self.workflow.builder.image_id = image_id
 
-        return image_id
+        media_types = ['application/json']
```

Comment:
In the near future, pulp_pull will need to move into the orchestrator build, and there it won't know whether pulp_pull ran. Maybe it should be told via parameter, based on what api versions osbs.conf says to support?